config/suricata/etc/suricata.yaml

```diff
@@ -39,44 +39,58 @@ vars:
     MODBUS_PORTS: 502
     FILE_DATA_PORTS: "[$HTTP_PORTS,110,143]"
     FTP_PORTS: 21
+    GENEVE_PORTS: 6081
     VXLAN_PORTS: 4789
-    
+    TEREDO_PORTS: 3544
+
 ##
-## Step 2: select outputs to enable
+## Step 2: Select outputs to enable
 ##
 
 # The default logging directory.  Any log or output file will be
-# placed here if its not specified with a full path name. This can be
+# placed here if it's not specified with a full path name. This can be
 # overridden with the -l command line parameter.
 default-log-dir: /var/log/suricata/
 
-# global stats configuration
+# Global stats configuration
 stats:
   enabled: no
-  # The interval field (in seconds) controls at what interval
-  # the loggers are invoked.
-  interval: 300
-  # Add decode events as stats.
+  # The interval field (in seconds) controls the interval at
+  # which stats are updated in the log.
+  interval: 8
+  # Add decode events to stats.
   #decoder-events: true
   # Decoder event prefix in stats. Has been 'decoder' before, but that leads
   # to missing events in the eve.stats records. See issue #2225.
   #decoder-events-prefix: "decoder.event"
   # Add stream events as stats.
   #stream-events: false
 
+# Configure the type of alert (and other) logging you would like.
 outputs:
+  # a line based alerts log similar to Snort's fast.log
   - fast:
-      enabled: no
+      enabled: yes
```

Comment:
What is it used for?

Reply:
fast.log contains alerts consisting of a single line similar to Snort's fast.log. It comes enabled in the default configuration of Suricata. So basically the output generated by Suricata are written to the  ```fast.log``` along with ```eve.json``` file.